.github/workflows/build-container.yml

```diff
@@ -137,12 +179,8 @@ jobs:
 
       - name: Start ceph cluster
         run: |
-          if ! docker-compose --version 2>&1 > /dev/null ; then
-            sudo apt update
-            sudo apt install -y docker-compose
```

Comment:
not must but better to add a failure catch, something like
```
sudo apt install -y docker-compose
if [ $? -ne 0 ]; then
  echo "Failed to install docker-compose" >&2
  exit 1
fi
```

Reply:
We don't need `docker-compose` v1 anymore. And the `docker compose` cmd is already installed in the runner.